src/libraries/System.ComponentModel.Composition/src/System/ComponentModel/Composition/ReflectionModel/ReflectionComposablePart.cs

```diff
@@ -300,7 +301,7 @@ private object GetInstanceActivatingIfNeeded()
                 }
 
                 // if the instance has been already set
-                if (createdInstance == null)
```

Comment:
I think this is bug, if `createdInstance` is null, no need to release it, CC @safern @maryamariyan